src/docs/index.md

```diff
@@ -21,7 +21,9 @@ description: The landing page for Flutter documentation.
 
 Flutter released [version
 1.2](https://developers.googleblog.com/2019/02/launching-flutter-12-at-mobile-world.html)
-at Mobile World Congress (MWC) in Barcelona!
+at Mobile World Congress (MWC) in Barcelona. For more information, see
+[Release Notes Flutter
```

Comment:
The name of the page is awkward (and the heading on the page has the opposite order). How about just plain text?

the [Flutter 1.2.1 release notes]
OR
the [release notes]